src/Uno.UI/UI/Xaml/Media/Animation/Storyboard.cs

```diff
@@ -119,15 +102,24 @@ public void Begin()
 
 		private void Play()
 		{
-			if (this.Children != null)
+			if (Children != null && Children.Count > 0)
 			{
-				foreach (ITimeline child in this.Children)
+				foreach (ITimeline child in Children)
 				{
 					_runningChildren++;
 					child.Completed += Child_Completed;
 					child.Begin();
 				}
 			}
+			else
+			{
+				Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
```

Comment:
why does this need to be dispatched? (And if it does, the 'immediately' in the comment isn't accurate)

Reply:
Using is not raising the `Completed` synchronously, so we're doing the same.

Here "Immediately" means we have nothing to wait for. But I agree it could be confusing.